build-aux/.run-format.zsh

```diff
@@ -50,8 +50,8 @@ invoke_formatter() {
         exit 2
       fi
 
-      local -a source_files=((libobs|libobs-*|UI|plugins)/**/*.(c|cpp|h|hpp|m|mm)(.N))
-      source_files=(${source_files:#*/(obs-websocket/deps|decklink/*/decklink-sdk|enc-amf|mac-syphon/syphon-framework|obs-outputs/ftl-sdk)/*})
+      local -a source_files=((libobs|libobs-*|UI|plugins|deps)/**/*.(c|cpp|h|hpp|m|mm)(.N))
```

Comment:
I was under the impression that the contents of the `deps` directory is vendored code that we don't expect to be formatted - has this changed?

Reply:
As an example `opt-parser` is not vendored code.

Reply:
Sounds good to me then.